### ISSUE TITLE
MeshCentral crashes when logging in after setting up 2FA

### ISSUE THREAD

**Issue by whalehub** [Ylianst/MeshCentral#86]
I'm using MeshCentral v0.2.7-p and set up TOTP on my administrator account just now. After logging out and then logging back in with my credentials, instead of being prompted for the TOTP code, I'm greeted by a blank page. The following error message appears in mesherrors.txt:

```
-------- 2/8/2019, 8:42:19 PM ---- 0.2.7-p --------

/meshcentral/node_modules/meshcentral/webserver.js:380
        if (domain.yubikey.id && domain.yubikey.secret && user.otphkeys && (user.otphkeys.length > 0) && (typeof (token) == 'string') && (token.length == 44)) {
                           ^

TypeError: Cannot read property 'id' of undefined
    at checkUserOneTimePassword (/meshcentral/node_modules/meshcentral/webserver.js:380:28)
    at /meshcentral/node_modules/meshcentral/webserver.js:434:21
    at /meshcentral/node_modules/meshcentral/webserver.js:251:32
    at AsyncWrap.<anonymous> (/meshcentral/node_modules/meshcentral/pass.js:31:88)
    at AsyncWrap.wrap.ondone (internal/crypto/pbkdf2.js:37:48)
```

**Comment by Ylianst** [comment]
OH!! Working on it now!!

**Comment by Ylianst** [comment]
Published v0.2.7-r to fix that. Can you give it a try? - Thanks!

**Comment by whalehub** [comment]
@Ylianst Thanks for the quick fix, the problem is gone now.
